### PR TITLE
Issue #70 - fix

### PR DESCRIPTION
**Pull request by maticko**
Fixed problem with issue #70 because models were defined in wrong place.

### CODE REVIEW

File: piplmesh/api/models.py [hunk 1]
```diff
@@ -1,3 +1,50 @@
-from django.db import models
+import datetime
 
-# Create your models here.
+import mongoengine
+
+from piplmesh.models import User
```
Comment (mitar) [inline piplmesh/api/models.py]: This is not correct. Wrong import. And import only modules!

Comment (maticko) [inline piplmesh/api/models.py]: Fixed?

**Comment by mitar** [comment]
You should add `piplmesh.api` to `INSTALLED_APPS` and you will see if it works. :-)

**Comment by maticko** [comment]
looks ok to me.

File: piplmesh/settings.py [hunk 2]
```diff
@@ -145,11 +145,12 @@
 
 INSTALLED_APPS = (
     # Ours are first so that we can override default templates in other apps
-    'piplmesh.frontend',
     'piplmesh.account',
-    
+	'piplmesh.api',
```
Comment (mitar) [inline piplmesh/settings.py]: Indentation!

Comment (maticko) [inline piplmesh/settings.py]: Fixed.

File: piplmesh/settings.py [hunk 3]
```diff
@@ -145,11 +145,12 @@
 
 INSTALLED_APPS = (
     # Ours are first so that we can override default templates in other apps
-    'piplmesh.frontend',
     'piplmesh.account',
-    
+    'piplmesh.api',
+    'piplmesh.frontend',
+
+	'django.contrib.messages',
```
Comment (mitar) [inline piplmesh/settings.py]: Indentation! Still!

**Comment by maticko** [comment]
This is fixed now....

**Pull request merged**